src/pages/trader-tools/_validation.js

```diff
@@ -25,7 +26,7 @@ const validation_is_not_zero = (input) => {
 
 const numberValidation = (input, fieldName, maxDigit) => {
     if (!input) {
-        return localize(`${fieldName} is required`)
+        return <Localize translate_text="{{fieldName}} is required" values={{ fieldName }} />
```

Comment:
```suggestion
        return <Localize translate_text="{{fieldName}} is required" values={fieldName} />
```

Reply:
The syntax should have double `{{` or else it will not extracted